app/timetables/tests/test_models.py

```diff
@@ -235,3 +237,49 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class EventTest(TestCase):
```

Comment:
Yes django save method works, but you would want to test normal usage to ensure that you are using the save method properly